chatkit-android/build.gradle

```diff
@@ -45,28 +45,23 @@ android {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     if (rootProject.properties.containsKey('pusher_platform_local')) {
         api project(':pusher-platform-android')
     } else {
         api "com.pusher:pusher-platform-android:$pusher_platform_version"
     }
     api project(':chatkit-core')
 
-    testImplementation "junit:junit:$junit_legacy_version"
-    testImplementation junit5.unitTests()
-    testCompileOnly junit5.unitTestsRuntime()
-    testImplementation junit5.parameterized()
-    testImplementation "org.mockito:mockito-inline:$mockito_version"
-    testImplementation 'com.google.truth:truth:0.40'
-
-    androidTestImplementation("com.android.support.test.espresso:espresso-core:$espresso_version", {
-        exclude group: 'com.android.support', module: 'support-annotations'
-    })
+    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
+    implementation 'com.pusher:push-notifications-android:1.6.2'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
It's good to keep dependencies in the order of importance, hence I would keep Kotlin stdlib-jdk7 at the top of the 3rd party dependencies.

Reply:
Makes sense - have put it at the top!

Reply:
Now it has disappeared altogether in this file. It's safer to explicitly declare it as it's being used in `chatkit-android`. Currently it's coming implicitly as transitional dependency of `chatkit-core`.

Reply:
This was a misunderstanding of https://github.com/pusher/chatkit-android/pull/169#discussion_r376477863 - have added it back 😅